src/tools/log-parser-rs/src/main.rs

```diff
@@ -0,0 +1,72 @@
+#![warn(unused_crate_dependencies)]
+#![warn(missing_debug_implementations)]
+#![warn(missing_docs)]
+
+//!`kata-log-parser` is a tool that combines logfiles generated by the various
+//!system components, sorts them by timestamp, and re-displays the log entries. A
+//!time delta is added to show how much time has elapsed between each log entry.
```

Comment:
I don't think we're handling the deltas yet right? If so, could you remove this and raise an issue so we don't forget.